packages/ui/src/components/molecules/EditableInput/EditableInput.tsx

```diff
@@ -41,6 +41,10 @@ const EditableInput = ({
   const [isEditable, setEditable] = useState(true);
   const [value, setValue] = useState(defaultValue);
 
+  useEffect(() => {
+    setValue(defaultValue);
+  }, [defaultValue]);
```

Comment:
I understand why you do that but I think it's better to do that in another ticket in dev

Reply:
i created a ticket, but i let it there so it works